pkgs/tools/misc/nncp/default.nix

```diff
@@ -9,24 +10,28 @@
 
 stdenv.mkDerivation rec {
   pname = "nncp";
-  version = "5.3.3";
+  version = "5.4.1";
 
   src = fetchurl {
     url = "http://www.nncpgo.org/download/${pname}-${version}.tar.xz";
-    sha256 = "1l35ndzrvpfim29jn1p0bwmc8w892z44nsrdnay28k229r9dhz3h";
+    sha256 = "0qzs4zkaj6qmg7ghqab4ba07p7w5qkw05q8pbh8zhgaka6dhqbd0";
   };
 
-  nativeBuildInputs = [ go ];
+  nativeBuildInputs = [ go redo-apenwarr ];
 
-  preConfigure = ''
+  buildPhase = ''
     export GOCACHE=$PWD/.cache
+    export CFGPATH=/etc/nncp.hjson
+    export SENDMAIL=/run/wrappers/bin/sendmail
+    redo ''${enableParallelBuilding:+-j''${NIX_BUILD_CORES}}
   '';
 
-  makeFlags = [
-    "PREFIX=${placeholder "out"}"
-    "CFGPATH=/etc/nncp.hjson"
-    "SENDMAIL=/run/wrappers/bin/sendmail"
-  ];
+  installPhase = ''
+    export PREFIX=${placeholder "out"}
```

Comment:
No need for `placeholder` here.

Either

```sh
export PREFIX=$out
```

Or

```nix
  PREFIX = placeholder "out";
```

(Either is acceptable. I personally prefer the former.)

Reply:
Thank you. I've changed this.